doc/pandoc.txt

```diff
@@ -1154,6 +1148,16 @@ A description of the available configuration variables follows:
 
   Use default mappings.
 
+BREAKING CHANGES                                        *vim-pandoc-changes*
+
+As discussed at https://github.com/vim-pandoc/vim-pandoc/issues/272 the chdir
+module, previously being enabled by default, was removed. Please add
```

Comment:
I think the suggestion in the issue is better than this. I also would prefer to have this in the commit description than in the documentation, where it will become outdated at some point.